### ISSUE TITLE
Undefined variable $page

### ISSUE THREAD

**Issue by skylennard** [hotmeteor/inertia-statamic#1]
I'm trying to implement this connector (v0.1.2) in my Statamic v3.1.23 site and I'm getting this `undefined variable $page` error.

When troubleshooting, I've discovered it's happening due to `$page->augmentedValue('template') === 'app'` being `false` (in InertiaStatamic.php), so the Inertia render isn't triggered. If I note out this conditional statement, then the page seems to load!

From what I can tell now, there's no template value being augmented anymore that indicates which template is rendering the requested path?

This is definitely confusing...

Help?

Thanks so much!

**Comment by hotmeteor** [comment]
What's your root page called? It should be `app.php`

**Comment by skylennard** [comment]
I'm not sure which page you're referring to...?

in `/server.php`, there's the Laravel standard `require_once __DIR__.'/public/index.php';`, does this need to be app.php instead?

(that index.php file is the one that comes packaged with Laravel)

The base view for this site is using `/views/app.blade.php`, and the base Vue application is at `/resources/js/App.vue`.

What am I missing?

**Comment by hotmeteor** [comment]
No, your Inertia root template. See https://inertiajs.com/server-side-setup

**Comment by skylennard** [comment]
So it needs to be `/views/app.php`, not `/views/app.blade.php`?

**Comment by hotmeteor** [comment]
Shoot sorry, no. It should be `/views/app.blade.php`.

**Comment by skylennard** [comment]
Ok cool, so yes that’s already set up…I’m pretty sure I’ve followed all the directions so it’s confusing for sure. When that condition in this library  is noted out, the page loads with the initial data attribute populated, and I assume it would work from there as expected…

**Comment by hotmeteor** [comment]
@skylennard Did you ever work this out?

**Comment by jonlemarquand** [comment]
Hi Adam,

Think I've at least managed to recreate this one, or something similar. Steps to recreate:

- New install of Statamic/Inertia (no content)
- Update page 'Home'
- Add template to 'blueprint' and set to 'app'

For whatever reason, Statamic was saving with template 'home' and template_field 'app'. Updating it in the .md file seemed to work.

<img width="423" alt="Screenshot 2022-05-06 at 20 46 09" src="https://user-images.githubusercontent.com/38497188/167207283-3faff65d-d9f1-49f8-963c-96c44f9cf74c.png">
